controllers/grid/LocaleGridRow.inc.php

```diff
@@ -35,7 +35,7 @@ function __construct($tabsSelector) {
 	/**
 	 * @copydoc GridRow::initialize()
 	 */
-	function initialize($request) {
+	function initialize($request, $template = NULL) {
 		parent::initialize($request);
```

Comment:
The `$template` variable should be passed into the call to `parent::initialize`.